EMML/1-1000/EMML101.xml

```diff
@@ -94,6 +96,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://ra
       <revisionDesc>
          <change who="PL" when="2017-11-30">Created minimal record with link from data sent from Daniel Gullo, HMML
                             project.</change>
+         <change when="2022-04-06" who="GS"></change> revised dimension
```

Comment:
the content must be between the open-tag and close-tag
`<open-tag>CONTENT</close-tag>`

Reply:
to correct this record you must open the [GST-202-0406] branch in your github desktop,
then open emml101.xml in your xml editor, carry out the corrections, save, commit, push.